lib/ueberauth/strategy/slack.ex

```diff
@@ -31,6 +31,9 @@ defmodule Ueberauth.Strategy.Slack do
     if conn.params["state"], do: opts = Keyword.put(opts, :state, conn.params["state"])
     callback_url = callback_url(conn)
 
+    team = get_config(:team)
```

Comment:
Nice, thanks. I thought something like this existed, but I couldn't find it. Fix coming up.